src/main/java/com/coutemeier/maven/plugins/asciidoctor/lifecycle/Messages.java

```diff
@@ -0,0 +1,38 @@
+package com.coutemeier.maven.plugins.asciidoctor.lifecycle;
+
+/**
+ * Messages to show in logs.
+ */
+public final class Messages {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Messages' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/asciidoctor-lifecycle-maven-plugin/asciidoctor-lifecycle-maven-plugin/pullRequest?prid=5396310)